Find a functionally equivalent code:android.app.Activity.onStop:COMMENT
Method Modifier: protected   
Comment:/**
 * Called when you are no longer visible to the user.  You will next
 * receive either {@link #onRestart}, {@link #onDestroy}, or nothing,
 * depending on later user activity.
 *
 * <p>Note that this method may never be called, in low memory situations
 * where the system does not have enough memory to keep your activity's
 * process running after its {@link #onPause} method is called.
 *
 * <p><em>Derived classes must call through to the super class's
 * implementation of this method.  If they do not, an exception will be
 * thrown.</em></p>
 *
 * @see #onRestart
 * @see #onResume
 * @see #onSaveInstanceState
 * @see #onDestroy
 */

@@ -3,8 +3,9 @@
         Slog.v(TAG, "onStop " + this);
     if (mActionBar != null)
         mActionBar.setShowHideAnimationEnabled(false);
+    mActivityTransitionState.onStop();
     getApplication().dispatchActivityStopped(this);
     mTranslucentCallback = null;
     mCalled = true;

